test/uk/gov/hmrc/plasticpackagingtax/registration/controllers/CheckLiabilityDetailsAnswersControllerTest.scala

```diff
@@ -93,6 +94,36 @@ class CheckLiabilityDetailsAnswersControllerTest extends ControllerSpec {
         )
       }
 
+      "group registration enabled and group of organisation is selected" in {
+        authorizedUser(features = Map(Features.isGroupRegistrationEnabled -> true))
+
+        val registration = aRegistration(withRegistrationType(Some(RegType.GROUP)))
+        mockRegistrationFind(registration)
+        given(page.apply(refEq(registration), any(), any())(any(), any())).willReturn(
+          HtmlFormat.empty
+        )
+        when(mockStartRegistrationController.startLink(any())).thenReturn(startLiabilityLink)
```

Comment:
Do we need these 2 mock behaviour setting calls as they are made in the `beforeEach()` method?

Reply:
As we need to set the registrationType to GROUP in registration , I tried without mock but got NPE, so had to mock the behaviour here